### PR TITLE
indexOf for arrays

### PR DESCRIPTION
**Pull request by Wouterz90**

### CODE REVIEW

File: dist/lualib/typescript.lua [hunk 1]
```diff
@@ -76,6 +76,15 @@ function TS_every(list, func)
     return #list == #TS_filter(list, func)
 end
 
+function TS_indexOf(list, object )
+    for i = 1, #list do
+        if object == list[i] then
+            return i
+        end
+    end
+    return nil
```
Comment (lolleko) [inline dist/lualib/typescript.lua]: Should return -1

**Comment by lolleko** [comment]
Could you add a test in lualib.spec.ts?

**Comment by Wouterz90** [comment]
I hope I'm doing it correctly, no idea what I'm doing actually.

**Comment by lolleko** [comment]
Almost, you can test your tests locally by runnin npm test. I can add/fix the tests later if you dont want to it. In that case hus remove the tes you added and then i will merge.

**Comment by Wouterz90** [comment]
Sadly I have to give up, thanks for bearing with me and I'll look at how it's done

**Comment by lolleko** [comment]
Added tests in https://github.com/Perryvw/TypescriptToLua/pull/52

**Pull request merged**